docs/interfaces/c-client-for-q.md

```diff
@@ -579,7 +579,7 @@ extern K sslInfo(K x); // returns an error if init fails, or a dict of settings
 if(handle==-3){K x=ee(sslInfo((K)0));printf("Init error %s\n",xt==-128?x->s:"unknown");r0(x);}
 ```
 
-A restriction for SSL/TLS connections – these can be used from the initialization thread only, i.e. the thread which first calls any `khp` function since the start of the application.
+A restriction prior to 4.1t 2023.11.10 for SSL/TLS connections – these can be used from the initialization thread only, i.e. the thread which first calls any `khp` function since the start of the application.
```

Comment:
confusing wording. Does this make more sense? Or did I misunderstand?

```suggestion
Prior to 4.1t 2023.11.10, SSL/TLS connections can be used from the initialization thread only, i.e. the thread which first calls any `khp` function since the start of the application.
```